features/prompt-registry.mdx

```diff
@@ -121,12 +129,36 @@ promptlayer.prompts.publish(
     prompt_template=pt,
 )
 ```
-They are also available in the following REST API endpoints 
+
+They are also available in the following REST API endpoints
 
 `/rest/get-prompt-template` ([read more](/reference/get-prompt-template))
 
 `/rest/publish-prompt-template` ([read more](/reference/publish-prompt-template))
 
+### Model Parameters
+
+You can also add model parameters to your prompt templates. This allows you to set the provider, model, temperature, max_tokens, and other parameters for each prompt template. This is useful if you want to use different parameters for different prompts.
```

Comment:
"You can also add model parameters to your prompt templates."

-->

"Custom model parameters can be associated with individual prompt templates versions.

Reply:
@jzone3 updated.